documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -3212,7 +3212,7 @@ You can find out more about Bugzilla at:
 == Phabricator
 
 The FreeBSD Project utilizes https://reviews.freebsd.org[Phabricator] for code review requests.
-See the https://wiki.freebsd.org/CodeReview[CodeReview] wiki page for details.
+See the https://wiki.freebsd.org/Phabricator[Phabricator] wiki page for details.
```

Comment:
I would change the link name to `Phabricator wiki page` to clarify that it's another address because of the other `Phabricator` link.